### ISSUE TITLE
Procedure to auto-terminate long-running queries on autoscaled Aurora instances

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/openFEC#4759]
**What we're after**
Procedure to auto-terminate long-running queries on autoscaled Aurora instances

**Why we're after it**
Currently long-running queries are only automatically terminating on the writer node and instances 1 and 2. In times of high load, CPU can build up on auto-scaled instances and overwhelm the system

**Completion criteria**
- [ ] Long-running queries are auto-terminated on autoscaled Aurora instances

**Comment by dzhang-fec** [comment]
Thanks!
working. For now it is under the control which is using DC4 server to kill them.
However, we are under the AWS migration. After this we will different approach. Is it OK? @lbeaufort

**Comment by dzhang-fec** [comment]
Finished:
Summary: Quick update on the auto kill autoscaling >5min website long query. The kill script has been scheduled to run every 10mins, after testing.

Problem: the read-cluster endpoint cannot be used to kill the auto added sessions, because we have to login into the new one. The read-cluster endpoint only have the replica1 and 2 but not has the new added one. I did many tests to compare the sql statements cross them, e.g., total count(*) in the reader ro endpoint hasn;t incl.  the new node.

How to find the new nodes name?  select *  from aurora_replica_status() where log_stream_speed_in_kib_per_second >0  and server_id like 'application-autoscaling%'
Also, Rohan added a new table for us to easily check the new one info,  select *  from public.aurora_xxx where available = 'YES'

Additionally, to monitor this autoscaling, we have created http://172.16.91.81:8430/  to show the autoscaling number, time and the details. Thanks a lot for Rohen.
![image](https://user-images.githubusercontent.com/62447258/107430017-a5ba1e00-6af2-11eb-9121-2a5e78e9cfbf.png)
